docs/learn/learn-comparisons-rollups.md

```diff
@@ -47,7 +47,7 @@ example, optimistic rollups often have challenge periods of several hours or eve
 A **non-interactive method** (associated with validity proofs) relies on cryptographic mechanisms,
 such as SNARKs or STARKs, to validate transactions upfront. These proofs are generated off-chain and
 submitted to L1 for verification without requiring dispute resolution. While these methods offer
-near-instant finality, in theory, in practice delays are sometimes intentionally introduced as
+near-instant finality (limited by L1's finality), in theory, in practice delays are sometimes intentionally introduced as
```

Comment:
In light of what was said, along with what I know - it seems that it depends on the protocol. Ultimately, a rollup is only as good as where it is deriving its security/finality from (perhaps with the exception of a sovereign rollup). Therefore, I'll suggest the following:

```suggestion
potential near-instant finality on the rollup itself, it is ultimately limited by L1's finality (unless it is a sovereign rollup, in which case it would vary). In practice, delays are sometimes intentionally introduced as
```